assets/schema_input.json

```diff
@@ -10,15 +10,19 @@
             "patient": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "ID name must be provided and cannot contain spaces"
+                "errorMessage": "Patient name must be provided and cannot contain spaces",
```

Comment:
```suggestion
                "errorMessage": "Patient ID must be provided and cannot contain spaces",
```

Reply:
Just me being paranoid, I know we all know that we don't want the actual name of patient here, but I'd rather we cover all our bases.
I'm even concerned that patient ID might be misinterpreted...

Reply:
Makes sense! Can also go back to "ID name", I didn't know the reason when I changed it